packages/react-core/src/components/Nav/NavExpandable.tsx

```diff
@@ -34,7 +36,7 @@ interface NavExpandableState {
 }
 
 export class NavExpandable extends React.Component<NavExpandableProps, NavExpandableState> {
-  static displayName = 'NavExpandable';
+  static displayName = 'NavExpandableItem';
```

Comment:
I think this class should be called `NavExpandable`. Why did you rename it to `NavExpandableItem`?

Reply:
Sorry, I didn't think that it will affect one something but component type. I just thought `NavExpandableItem` is a bit more explicit than `NavExpandable`.

Reply:
Yes I agree our name should be something like `NavItemExpandable` and we can change it to that next breaking change release. In the meantime changing just the displayName will cause confusion with devs since the class name and file name are just `NavExpandable` and we can't change that.

Reply:
Gotcha, thanks for the explanation. I'll revert my changes here.